build.override.targets

```diff
@@ -9,6 +9,13 @@
     </TraversalBuildDependsOn>
   </PropertyGroup>
   
+    <PropertyGroup>
+    <TraversalCleanDependsOn>
+      WcfClean;
```

Comment:
In line 8, we have `WcfCleanup` and we have `WcfClean` here, do we really need 2 different targets for "clean"? Can we have a better solution?